--- a/test/cli/test_commands.py
+++ b/test/cli/test_commands.py
@@ -12,7 +12,7 @@
 import hypothesis
 import pytest
 import requests
-import trustme
+# import trustme
 import urllib3.exceptions
 import yaml
 from _pytest.main import ExitCode
@@ -213,6 +213,7 @@
     assert cli.run(schema_url, "--force-color", "--no-color") == snapshot_cli
 
 
+@pytest.mark.skip(reason = "cryptography broken atm")
 @pytest.mark.openapi_version("3.0")
 @pytest.mark.operations("success")
 def test_certificates(cli, schema_url, mocker):
